src/main/java/graphql/schema/SchemaUtil.java

```diff
@@ -17,23 +17,26 @@
 
 
     Map<String, GraphQLType> allTypes(final GraphQLSchema schema, final Set<GraphQLType> additionalTypes) {
-        List<GraphQLType> roots = new ArrayList<GraphQLType>() {{
-            add(schema.getQueryType());
+        List<GraphQLType> roots = new ArrayList<>();
```

Comment:
The anon class keeps a reference to the outer instance, which is irrelevant here but I figured it might become relevant as the code gets refactored in the future. This is the only reason I removed the anon class.